### PR TITLE
chore: added date formats for reuse

### PR DESCRIPTION
**Pull request by madhuravius**
Full format (in UTC):

image

Example tooltip:

image

---

We can use these as needed in various places

### CODE REVIEW

**Comment by joeclayallday** [comment]
I pushed a change to stop this:

BEFORE
Screen Shot 2023-07-12 at 8 35 42 PM


AFTER
Screen Shot 2023-07-12 at 8 48 49 PM

I'm not sure if we need the autoSizeWidth stuff now.

**Review by joeclayallday — DISMISSED** [review]

File: index.css [hunk 1]
```diff
@@ -27,6 +27,7 @@ html, body, #app {
 .tooltip-inner {
   display: none;
   transform: translate(0%, -110%);
+  width: max-content;
```
[the remark this answers is not in the document]
Comment (joeclayallday) [reply index.css:30]: K, I don't think we need to worry too much about tooltips on smaller screens. Another approach is to just make a parent class that i can use specifically for the top-level page explainer tooltips.

.tooltip-format {
 .tooltip-inner {
   width: max-content;
}
}

[the remark this answers is not in the document]
Comment (madhuravius) [reply index.css:30]: what's strange to me is tailwind should have classes for these (max-content) for example, but i'm guessing we don't use this class directly in the code?

[the remark this answers is not in the document]
Comment (joeclayallday) [reply index.css]: Screen Shot 2023-07-13 at 9 22 02 AM
Screen Shot 2023-07-13 at 9 22 07 AM

File: yarn.lock [hunk 2]
```diff
@@ -5921,14 +5921,14 @@ __metadata:
   linkType: hard
 
 "tough-cookie@npm:^4.1.2":
```
Comment (madhuravius) [inline yarn.lock:5923]: this is weird, shouldn't be in the diff

File: index.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (madhuravius) [reply index.css]: with changes that i put in, basically i think the same thing?

image

image

File: yarn.lock (hunk 2 above)
Comment (joeclayallday) [reply yarn.lock]: Thank you!

**Pull request merged**